images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -62,6 +63,71 @@ function Install-Binary
     }
 }
 
+Function Install-VS
+{
+    <#
+    .SYNOPSIS
+        A helper function to install Visual Studio.
+
+    .DESCRIPTION
+        Prepare system environment, and install Visual Studio bootstrapper with selected workloads.
+
+    .PARAMETER BootstrapperUrl
+        The URL from which the bootstrapper will be downloaded. Required parameter.
+
+    .PARAMETER BootstrapperName
+        The Name with which bootstrapper will be downloaded. Required parameter.
+
+    .PARAMETER WorkLoads
+        The string that contain workloads that will be passed to the installer.
+    #>
+  Param
+  (
+    [Parameter(Mandatory)]
+    [String] $BootstrapperName,
```

Comment:
Can we extract `$BootstrapperName` from `$BootstrapperUrl` like
```
$BootstrapperName = [IO.Path]::GetFileName($BootstrapperUrl)
```

Reply:
Good point, thanks!